xcube/core/resampling/spatial.py

```diff
@@ -196,7 +196,7 @@ def resample_in_space(
             )
             downscaled_dataset = resample_dataset(
                 source_ds,
-                ((x_scale, 1, 0), (1, y_scale, 0)),
```

Comment:
I came across this through an example and realized that it should be inverted. It is not covered by any test. 

The code applies these steps if `0.95 target_gm.x_res > source_gm.x_res`, meaning that the source has a finer resolution than the target resolution. If this is the case the source dataset shall be resampled to a coarser grid, which has a similar resolution like the target grid. But since `x_scale < 0.95`, the grid will be even finer. However, the inverse downsamples the source dataset to the desired resolution.

Reply:
However, I forot to change it in the `if source_gm.is_regular:`, which I will correct for in the next push

Reply:
I think line 184 (`            downscaled_gm = source_gm.scale((x_scale, y_scale))
`) also needs this inversion.